pkg/urbit/include/noun/aliases.h

```diff
@@ -107,3 +107,14 @@
 #     define  u3tn(type, x) (x == 0) ? (void *)0 :  ((type *) u3a_into(x))
 #     define  u3of(type, x) (u3a_outa((type *)x))
 
+
+/* turn off address sanitizer to increase speed
+   https://en.wikipedia.org/wiki/AddressSanitizer
+*/
+#if GCC
+	/* gcc style */
+    #define NO_SANITIZE_ADDRESS  __attribute__((no_sanitize_address))
+#else
+    /* Clang style */
+    #define NO_SANITIZE_ADDRESS  __attribute__((no_sanitize("address")))
+#endif
```

Comment:
I've tested locally, and the GCC style is also supported by clang. Sorry for the misdirection, these definitions can be removed and the GCC style can be used directly in imprison.c.